### DISCUSSION TITLE
Institute verification

### DISCUSSION THREAD

**Discussion by beatrasbe** [RosettaCommons/rosetta#79] [ROSIE] [answered]
Hello,
I'd like to access ROSIE but I'm getting an error that I have to confirm my institution is academic. I currently work at Bizkaia Research Health Institute (https://www.biocrucesbizkaia.org/en/web/guest/iis). Thus, it would be great if both got marked as academic institutions.
Thank you and have a nice day,
Beatriz

**Comment by lyskov** [comment]
@beatrasbe Could you please list domains which your institute use? Also, could you please confirm that this is non-profit/academic org? (it is really not clear from web site)
Thanks,

**Reply by beatrasbe** [reply]
@lyskov. The domains used are bio-bizkaia.eus or osakidetza.eus. We are a non-profit organisation. It is a Health Research Institute. Please let me know if you need any aditional information. Thanks!

**Comment by lyskov** [comment, marked as answer]
This is now should be fixed. Please feel free to re-open if you still having trouble logging-in. Thanks,
